docs/en_US/builtin_assessor.rst

```diff
@@ -1,6 +1,16 @@
 Builtin-Assessors
 =================
 
+In order to save our computing resources, NNI supports an early stop policy and creates **Assessor** to finish this job.
+
+Assessor receives the intermediate result from Trial and decides whether the Trial should be killed by specific algorithm. Once the Trial experiment meets the early stop conditions(which means assessor is pessimistic about the final results), the assessor will kill the trial and the status of trial will be `"EARLY_STOPPED"`.
+
+Here is an experimental result of MNIST after using 'Curvefitting' Assessor in 'maximize' mode, you can see that assessor successfully **early stopped** many trials with bad hyperparameters in advance. If you use assessor, we may get better hyperparameters under the same computing resources.
```

Comment:
>bad hyperparameters 

under performed hyperparameters 

> If you use assessor, we may get better hyperparameters under the same computing resources.

With Assessor enabled, we are more likely to get better hyperparameters under the same computing resources or time spent restrictions.